### DISCUSSION TITLE
How to prevent vite from reloading the app while I work on tests and run vitest?

### DISCUSSION THREAD

**Discussion by mitar** [vitest-dev/vitest#2095] [Q&A]
So I am running vite in one terminal and vitest in another terminal. Every time I edit `foo.test.ts` file, vite hot-reloads the app as well. But I would assume that it should not because nothing really changed for the app (but `foo.test.ts` is alongside `foo.ts` file which is used by the app). Is there a way to prevent vite from reloading on tests?

**Comment by alanorozco** [comment]
Exclude the test path pattern using [`server.watch.ignored`](https://vitejs.dev/config/server-options.html#server-watch).
